web/modules/custom/dpl_event/src/Plugin/rest/resource/v1/EventsResource.php

```diff
@@ -221,30 +222,41 @@
  *   }
  * )
  */
-final class EventsResource extends ResourceBase {
+final class EventsResource extends EventResourceBase {
 // phpcs:enable Drupal.Files.LineLength.TooLong
 
   /**
-   * {@inheritdoc}
+   * GET request: Get all eventinstances, hopefully cached.
    */
-  public static function create(ContainerInterface $container, array $configuration, $plugin_id, $plugin_definition): self {
-    /** @var mixed[] $serializer_formats */
-    $serializer_formats = $container->getParameter('serializer.formats');
-    return new self(
-      $configuration,
-      $plugin_id,
-      $plugin_definition,
-      $serializer_formats,
-      $container->get('logger.factory')->get('rest'),
-    );
-  }
+  public function get(): Response {
+    // Entity query, pulling all eventinstances.
+    $storage = $this->entityTypeManager->getStorage('eventinstance');
+    $ids = $storage->getQuery()
+      ->accessCheck(FALSE)
+      ->condition('status', TRUE)
+      ->execute();
 
-  /**
-   * Responds to GET requests.
-   */
-  public function get(): ResourceResponse {
-    // @todo Implement me
-    return new ResourceResponse([]);
+    $event_responses = [];
+
+    foreach ($ids as $id) {
+      $event_instance = $storage->load($id);
+
+      if ($event_instance instanceof EventInstance) {
+        $event_responses[] = $this->mapper->getResponse($event_instance);
+      }
+    }
+
+    $event_responses = $this->serializer->serialize($event_responses, 'application/json');
```

Comment:
```suggestion
    $event_responses = $this->serializer->serialize($event_responses, $this->serializerFormat($request));
```

This is the same approach we use elsewhere instead of hard coding the type